chart/templates/etcd-headless-service.yaml

```diff
@@ -1,6 +1,5 @@
 {{- if not .Values.experimental.isolatedControlPlane.headless }}
 {{- if or .Values.controlPlane.backingStore.etcd.deploy.enabled (include "vcluster.etcd.embedded.migrate" .) }}
-{{- if .Values.controlPlane.backingStore.etcd.deploy.headlessService.enabled }}
```

Comment:
Whats the reasoning behind removing this option? Sounds like that was not interfering with what you are trying to change.

Reply:
This was removed to prevent someone from manually disabling it and thus causing the headless service to not be created and subsequently hampering the etcd statefulset's working. We discussed it [here](https://github.com/loft-sh/vcluster/pull/2297#discussion_r1858750161).